### PR TITLE
adding morphology stack one package at a time: part 1

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pyquaternion/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+from spack import *
+
+
+class PyPyquaternion(PythonPackage):
+    """Python morphology manipulation toolkit"""
+
+    homepage = "https://kieranwynn.github.io/pyquaternion/"
+    url      = "https://files.pythonhosted.org/packages/ae/c8/02b30c4a86744d2e15f7f16ab353f7231bd0241117713e5d60f466044994/pyquaternion-0.9.5.tar.gz"
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-pyquaternion/package.py]: NO!
```suggestion
    url      = "https://pypi.org/packages/source/p/pyquaternion/pyquaternion-0.9.5.tar.gz"
```

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-pyquaternion/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Lawrence Livermore National Security, LLC and other
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-pyquaternion/package.py]:
```suggestion
# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-plotly-helper/package.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Lawrence Livermore National Security, LLC and other
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-plotly-helper/package.py]:
```suggestion
# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

**Review by matz-e — CHANGES_REQUESTED** [review]

**Comment by wizmer** [comment]
@matz-e Can we merge this first part please

**Review by pramodk — APPROVED** [review]
LGTM

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-cut-plane/package.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyCutPlane(PythonPackage):
+    """Python morphology manipulation toolkit"""
```
Comment (mgeplf) [inline var/spack/repos/builtin/packages/py-cut-plane/package.py:11]: c&p from py-morph-tool?

**Pull request merged**